ingest/common/domain/block_pools.go

```diff
@@ -1,4 +1,4 @@
-package commondomain
+package domain
```

Comment:
Why do we do this change? Would prefer the old naming as to avoid having multiple packages having naming clashes during imports. `domain` package exists across `common`, `sqs` and `indexer`

Reply:
Hey @p0mvn, this change was made by my [recommendation](https://github.com/osmosis-labs/osmosis/pull/8744#pullrequestreview-2347329639). Based on that suggestion, do you still prefer to keep old naming?

Reply:
That makes sense. Not a strong opinion and happy to proceed with what we have

Reply:
Although it is not strictly required, IMHO, I also think it is more conventional and good practice for the package name to be same as the folder name. So I'd go ahead and make this change. Thanks guys for the suggestions and opinions.